app/main.ts

```diff
@@ -1,7 +1,15 @@
 import { bootstrap }    from '@angular/platform-browser-dynamic';
-
+import { HTTP_PROVIDERS } from "@angular/http";
 import { AppComponent } from './app.component';
-
 import { appRouterProviders } from "./app.routes";
 
-bootstrap(AppComponent,[appRouterProviders]);
+
```

Comment:
These changes seems to be to communicate with an API. That's great!

Can you please let me know how it works? I would like to know the meaning of each line of code.

Thanks